.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,32 @@
+---
+name: 'Bug Report'
+about: 'Report a bug, something does not work as it supposed to'
+title: '[bug] SHORT DESCRIPTION'
+labels: bug
+---
+
+<!--
+  Please don't forget to update the issue title.
+  Include all applicable information to help us reproduce your problem.
+
+  To help us debug your issue please explain:
+-->
+
+### Environment Details (include every applicable attribute)
+  * Operating System+version: **Linux Ubuntu 18.04**
+  * Compiler+version: **GCC 8**
```

Comment:
Do not use concrete values or no-one is going to change them and it will be double confusing.

Reply:
done